.github/workflows/publish-pages.yml

```
@@ -0,0 +1,23 @@
+name: Publish pages folder to gh-pages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

```suggestion
          node-version: 18.13.0
```
`18.16.0` uses a different major version of `npm` than `18.13.0`